src/ParseObject.js

```diff
@@ -935,10 +939,22 @@ class ParseObject {
   }
 
   /**
-   * Clears any changes to this object made since the last call to save()
+   * Clears any (or specific) changes to this object made since the last call to save()
+   * @param {String|Array<String>|...String} [keys] - specify which fields to revert
    */
-  revert(): void {
-    this._clearPendingOps();
+  revert(...keys?: Array<string>): void {
+    let _keysToRevert;
+    if(keys.length) {
+      _keysToRevert = [];
+      keys.forEach((key) => {
+        if (Array.isArray(key)) {
```

Comment:
I’m not sure why it would support nested arrays.

Reply:
Me neither, but I'm following the pattern seen in ```Parse.Query#select|include```, e.g. https://github.com/parse-community/Parse-SDK-JS/blob/master/src/ParseQuery.js#L1374

Reply:
The big difference is the type definition of the keys property. It’s not the same in the 2 implementations